api/src/pcapi/connectors/clickhouse/queries/count_bookings.py

```diff
@@ -0,0 +1,49 @@
+import pydantic.v1 as pydantic_v1
+
+from pcapi.connectors.clickhouse.queries.base import BaseQuery
+
+
+class CountIndividualBooking(pydantic_v1.BaseModel):
```

Comment:
Par logique et par cohérence avec `CoutOffersQuery` j'ajouterais bien un `s`, ici et dans les classes en-dessous.
```suggestion
class CountIndividualBookings(pydantic_v1.BaseModel):
```